source/slang/diff.meta.slang

```diff
@@ -70,22 +70,34 @@ struct DifferentialPair : IDifferentiable
     __intrinsic_op($(kIROp_MakeDifferentialPair))
     __init(T _primal, T.Differential _differential);
 
-    __intrinsic_op($(kIROp_DifferentialPairGetDifferential))
-    T.Differential d();
+    property p : T
+    {
+        __intrinsic_op($(kIROp_DifferentialPairGetPrimal))
+        get;
+    }
+
+    property v : T
```

Comment:
Is `.v` used anywhere? It seems to be identical to `.p`

Reply:
They are not, I was at a point think about using `v` instead of `p` since `p` could mean `ptr` in many naming conventions.